tests/org.eclipse.n4js.expectmatrix.tests/xpect-tests/builtin_js/array/array_concat.n4js.xt

```diff
@@ -38,7 +38,10 @@ console.log(numeric);
 <==
 stdout:
 [ 3, 4, 5, 1, 2, 3 ]
-[ 1, 2, 3, 4, 5, 6, 7, 8, 9 ]
+[
```

Comment:
Should we use a JSON parser in the tests to assert the output?

Reply:
The output is not always valid JSON. But it might be an option in some cases (i.e. not changing output Xpect method in general but introducing a separate Xpect method that compares output via JSON).